documentation/source/user_section/tutorials/spinalcord_segmentation.rst

```diff
@@ -0,0 +1,243 @@
+   :align: right
+   :figwidth: 20%
+
+   3D mesh propagation using PropSeg
+
+``sct_propseg`` itself is a single command, but internally it uses three processing steps to segment the spinal cord.
+
+#. Detect the approximate center of the spinal cord automatically using a machine learning-based method (OptiC). This is an initialization step for the core algorithm, PropSeg.
+#. Create a coarse 3D mesh by propagating along the spinal cord (PropSeg).
+#. Refine the surface of the mesh using small adjustments.
+
+   .. note::
+
+      The centerline detection step is also provided in a standalone script called ``sct_get_centerline``.
+
+Example: T2w image
+------------------
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_image.png
+  :align: right
+  :figwidth: 10%
+
+  t2.nii.gz
+
+From the ``single_subject/data`` folder, navigate to the ``t2`` directory, and verify that it contains a single T2-weighted image:
+
+.. code:: sh
+
+   $ cd t2
+   $ ls
+   t2.nii.gz
+
+Running the script
+^^^^^^^^^^^^^^^^^^
+
+Next, run the following command to process the image:
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_qc.png
+  :align: right
+  :figwidth: 40%
+
+  The QC report for the segmented image
+
+.. code:: sh
+
+   $ sct_propseg -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
+
+Note that we have provided three arguments:
+
+- ``-i``, which indicates the input image (``t2.nii.gz``).
+- ``-c``, which indicates the contrast of the image (``t2``).
+- ``-qc``, the directory for Quality Control reporting (``~/qc_singleSubj``). QC reports will allow us to evaluate the segmentation slice-by-slice.
+
+During execution, the script will provide status updates as it progress through its various stages.
+
+Inspecting the results using QC
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When complete, the script will output a command to inspect the results. (**Note:** The exact filepath will vary depending on your filesystem.)
+
+.. code:: sh
+
+   Use the following command to see the results in a browser:
+   xdg-open "sct_course_london20/single_subject/data/t2/qc_singleSubj/index.html"
+
+Running this command in your Terminal window will open up a page in your default browser. On this page, the spinal cord is displayed slice by slice. It has also been cropped from the overall anatomical image to provide a quick overview. The segmentation is displayed using a red overlay that can be toggled by repeatedly pressing the right arrow key. More information about QC reporting can be found on the <link to QC reporting> page.
+
+Inspecting the results using FSLEyes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have fsleyes installed, the script will also output a second command to inspect the results. This will be true for all commands run in this tutorial. (**Note:** The exact filepath will vary depending on your filesystem.)
```

Comment:
```suggestion
If you have `FSLeyes <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes>`_ installed, the script will also output a second command to inspect the results. This will be true for all commands run in this tutorial. (**Note:** The exact filepath will vary depending on your filesystem.)
```